torchaudio/prototype/pipelines/source_separation_pipeline.py

```diff
@@ -65,8 +67,11 @@ def get_model(self) -> torch.nn.Module:
     _sample_rate=8000,
 )
 CONVTASNET_BASE_LIBRI2MIX.__doc__ = """Pre-trained ConvTasNet pipeline for source separation.
+
     The underlying model is constructed by :py:func:`torchaudio.prototyoe.models.conv_tasnet_base`
-    and utilizes weights trained on Libri2Mix using training script ``lightning_train.py``
-    `here <https://github.com/pytorch/audio/tree/main/examples/source_separation/>`__ with default arguments.
+    and utilizes weights trained on *Libri2Mix dataset* [:footcite:`cosentino2020librimix`] using training script
+    ``lightning_train.py`` `here <https://github.com/pytorch/audio/tree/main/examples/source_separation/>`__
```

Comment:
the link to main branch can be changed anytime, perhaps points to the latest release?